### ISSUE TITLE
Variable watchers don't adjust to zoom well

### ISSUE THREAD

**Issue by ToonTalk** [jmoenig/Snap--Build-Your-Own-Blocks#2103]
At zoom = 1 I created two variables. When I switched to zoom = 2 I see

![image](https://user-images.githubusercontent.com/2138696/40621946-16e7db9c-6297-11e8-8159-ef512b6922bd.png)

Creating the variables at 2x works fine. But saving and loading the above doesn't fix it.

**Comment by jmoenig** [comment]
Yep, you're right. Watchers don't zoom along with the stage, but they *do* zoom along with the blocks. This is a known issue and on my list.

**Comment by cycomachead** [comment]
> Watchers don't zoom along with the stage, but they do zoom along with the blocks. 

I'm not sure zooming with the stage is always what's wanted -- in full screen mode, the watchers are the same size, they just take up less of the stage. 

I've always liked that watchers resize with zoom blocks. it's useful, especially when presenting and since watchers often are meant to be read, it makes sense that the font size should be similar. 

I think the only thing that needs to happen is for them to be "reflowed" but this isn't easy if you've repositioned a watcher yourself.
